### DISCUSSION TITLE
How to set hyperparameters more reasonably?

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/deepmd-kit#555] [Q&A]
In the default example, "model": { "neuron": [25, 50, 100]...}, "fitting_net" : {"neuron": [240, 240, 240]...}. Is this default setting of hidden layers and neurons sufficient for general systems? How to set hyperparameters more reasonably?

**Comment by amcadmus** [comment]
These settings are usually sufficient for most of the systems. 

In any case you find the size of network is not large enough, i.e. the accuracy can be improved by using larger networks, you are welcome to report.
